lib/PreconditionRunner.pm

```diff
@@ -95,12 +96,53 @@ sub write_num_passes
 sub run_to_steady_state($)
 {
     my $self = shift;
-    
     my %args = @_;
 
     my $msg_prefix = $args{'msg_prefix'};
-    my $output_dir = $args{'output_dir'};
     my $test_ref = $args{'test_ref'};
+    my $output_dir = $args{'output_dir'};
+
+    # preconditioner.exe does not support multiple files
```

Comment:
If I understand correctly, the PreconditionRunner::run_to_steady_state() no longer always calls preconditon.exe and no longer runs to steady-state.  This is going to feel like giant tech debt some day.  Do words mean nothing?  Today FooRunner literally always runs Foo, right?

Why not just run two copies of precondition.exe simultaneously?

Or, if you absolutely must use diskspd for this case, maybe it's better to do it one layer above in the caller?